sources/HUBViewControllerImplementation.m

```diff
@@ -1205,8 +1198,37 @@ - (BOOL)performActionForTrigger:(HUBActionTrigger)trigger
                                                                                        viewModel:viewModel
                                                                                   componentModel:componentModel
                                                                                   viewController:self];
-    
-    return [self.actionHandler handleActionWithContext:context];
+
+    BOOL isActionHandled = [self.actionHandler handleActionWithContext:context];
```

Comment:
Nitpick, since this is in the past, it should be `actionWasHandled` (or similar)